src/services/api.js

```diff
@@ -35,14 +29,41 @@ export const initializeApi = ({
     });
 };
 
+const sendRequest = (defaultHeaders, method, body, headers, uri) => {
+  switch (method.toLowerCase()) {
```

Comment:
maybe method.toLowerCase() could be protected and used in a const. For example: const methodLowerCase = (method || '').toLowerCase()